Scripts/Python/grsnWallPython.py

```diff
@@ -684,6 +684,8 @@ def OnTimer(self,id):
         avatar = PtGetLocalAvatar()
         if(id == kWearOriginalClothes and wornItem):
             for item in wornItem:
+                if item[1] == 8:
```

Comment:
What does this magic number signify?

Reply:
Accessories, can give it a named variable